### PR TITLE
zip_creator

### PR DESCRIPTION
**Pull request by krassovki**
- Added main
- Added tests

### CODE REVIEW

File: Misha/Files/zip_creator/zip_test.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from zip_main import check_set_of_paths, create_archive
+import zipfile
+
+
+class ZipTest(unittest.TestCase):
+
+    def test_check_set_of_paths_with_wrong_paths(self):
+        set_of_paths = ['path/to/file.1', 'path.txt', 'file.1']
+        with self.assertRaises(FileNotFoundError) as context:
+            self.assertRaises(FileNotFoundError, check_set_of_paths(set_of_paths))
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_test.py]: You have double assertion here. Please remove one

File: Misha/Files/zip_creator/zip_main.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            modified_set.append(set_of_paths[index])
+    if len(modified_set) == 0:
+        raise FileNotFoundError('List of paths is empty')
+    else:
+        return set_of_paths
+
+
+def create_archive(set_of_paths, archive_name):
+    set_of_paths = check_set_of_paths(set_of_paths)
+    print(set_of_paths)
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_main.py:30]: Lets avoid print function and use logger instead 
https://docs.python.org/3/howto/logging.html

File: Misha/Files/zip_creator/zip_main.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    with ZipFile(archive_name, "w") as newzip:
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_main.py]: newzip is not python naming for variable

File: Misha/Files/zip_creator/zip_main.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        for path_to_file in set_of_paths:
+            newzip.write(path_to_file)
+    print('Archive created:', archive_name)
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_main.py]: Do not use print. Use logger instead

File: Misha/Files/zip_creator/zip_main.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
+from zipfile import ZipFile
+import os
+
+
+def input_set_of_files():
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_main.py:6]: Please add tests for this function

File: Misha/Files/zip_creator/zip_main.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    set_of_paths = []
+    input_string = ' '
+    while input_string != '':
+        input_string = str(input('Enter path to file: '))
+        set_of_paths.append(input_string)
+    return set_of_paths
+
+
+def check_set_of_paths(set_of_paths):
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_main.py:15]: Please add tests for this function

File: Misha/Files/zip_creator/zip_test.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        exception_message = str(context.exception)
+        self.assertEqual('List of paths is empty', exception_message)
+
+    def test_create_archive_with_right_set_of_paths(self):
+        set_of_paths = ['zip_main.py', 'zip_test.py']
```
Comment (glock3) [inline Misha/Files/zip_creator/zip_test.py:24]: Please create new folder with name 'test-resources' add some files there and archive them. You can add them to this commit 
You can use any file (picture, other archive, text file etc.)
Do not forget to clean this dir after tests. Try to find some 'after' phase for tests in the doc

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Misha/Files/zip_creator/zip_main.py (hunk 3 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_main.py]: i will correct

File: Misha/Files/zip_creator/zip_test.py (hunk 1 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_test.py]: i will correct

File: Misha/Files/zip_creator/zip_main.py (hunk 2 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_main.py:30]: done

File: Misha/Files/zip_creator/zip_main.py (hunk 3 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_main.py]: corrected

File: Misha/Files/zip_creator/zip_test.py (hunk 1 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_test.py]: corrected

File: Misha/Files/zip_creator/zip_main.py (hunk 4 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_main.py]: corrected

File: Misha/Files/zip_creator/zip_main.py (hunk 6 above)
Comment (krassovki) [reply Misha/Files/zip_creator/zip_main.py:15]: done

**Review by glock3 — APPROVED** [review]

**Pull request merged**